contribute/contribute-pull-requests/add_hook.md

```diff
@@ -0,0 +1,18 @@
+# How to add a new Hook
+Adding a new hook is a process that can be done in only three steps.
+
+## 1) Add a call to Hook::exec()
+
+As you can see on this [PR](https://github.com/PrestaShop/PrestaShop/pull/34431/files), adding a simple Hook::exec() statement is enough to add a new hook. In order for it to be complete, there is a couple more steps to do.
+
+## 2) Modify the hook.xml file
+
+In order to properly register the hook you need to add it to the hook.xml file as seen in this [PR](https://github.com/PrestaShop/PrestaShop/pull/34431/files)
```

Comment:
```suggestion
The second step is correctly adding the `Hook` during installation. You need to add it to the `hook.xml` file, as seen in this [PR](https://github.com/PrestaShop/PrestaShop/pull/34431/files). This will ensure that the `Hook` is added to the database, which enables users to sort modules hooked into it. It won't be possible without the hook in the database.
```